code/uri/1001.rb

```diff
@@ -0,0 +1,4 @@
+a = gets.to_i
+b = gets.to_i
+x = a +b
```

Comment:
Para manter a formatação sugerida pela comunidade ruby:
```suggestion
x = a + b
```